### ISSUE TITLE
`monaco is not defined` on CP CKEditor configs

### ISSUE THREAD

**Issue by realjoshharrison** [craftcms/ckeditor#150]
### Description

Hi! Small javascript error when defining CKEditor configs in the CP. `monaco is not defined`. This seems to have stopped some features of the config editor fields from working properly, e.g. autocompletion on the JSON mode, and config text field auto-sizing.

### Steps to reproduce

1. Clean Craft install, install CKEditor plugin
2. Go to define a config in `settings/ckeditor` 
3. Look at dev console – a javascript reference error is thrown.

### Additional info

- Craft version: 4.5.11.1 Pro
- PHP version: 8.2.0
- Database driver & version: MySQL 5.7.39
- Plugins & versions: CKEditor 3.6.0, CP Field Inspect 1.4.4

Thanks for taking a look :)

**Comment by justinholtweb** [comment]
+1 I am experiencing this also on a brand new install of CK Editor. 

To note: this is a multisite and I am also getting an additional JS error:
```
linkui.js:40 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'join')
    at a.value (linkui.js:40:48)
``` 

Attaching a screenshot of the editor screen

![Screen Shot 2023-11-30 at 11 17 59 AM](https://github.com/craftcms/ckeditor/assets/295903/9a586ba7-be08-4ebd-9baf-00779ebce121)

**Comment by i-just** [comment]
Hi, thanks for getting in touch! 

Looks like this is fixed in the latest version of `nystudio107/craft-code-editor` (see changelog: https://github.com/nystudio107/craft-code-editor/blob/develop/CHANGELOG.md#1016---20231130).

It worked as expected up to and including `1.0.13` and is fixed in `1.0.16`, but `1.0.16` is not yet available on Packagist.

You can temporarily fix this by requiring `"nystudio107/craft-code-editor": "1.0.13"` or `"nystudio107/craft-code-editor": "dev-develop#58e469ba2c2798150706bd11ae7fa6b854006208 as 1.0.15"` in your `composer.json` and running `composer update`.

(I also raised a PR to update the composer requirement for the CKEditor plugin to skip over versions `1.0.14` and `1.0.15`.)

**Comment by realjoshharrison** [comment]
Hi Iwona, thank you for taking a look so quickly, and the explanation!

I'll wait for the update, and use your fix if I run into problems 🙂

**Comment by realjoshharrison** [comment]
Hi @i-just - I have added `"nystudio107/craft-code-editor": "1.0.13"` to my `composer.json`and updated, which all works as expected. Can I ask whether I need to do remove it again once the proper fix ships?

**Comment by i-just** [comment]
@realjoshharrison, yes, it would be best to remove it then.

**Comment by khalwat** [comment]
This issue should be closed, it's already been addressed downstream in `craft-code-editor` https://github.com/nystudio107/craft-code-editor/commit/dea2736e60916cd309c68ae82accb30a5b5af339

**Comment by khalwat** [comment]
> It worked as expected up to and including 1.0.13 and is fixed in 1.0.16, but 1.0.16 is not yet available on Packagist.

I didn't tag that release properly, thanks for the heads up. I've re-tagged it, we should be good to go.
